pkgs/development/python-modules/dwdwfsapi/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, requests
+, ciso8601
+}:
+
+buildPythonPackage rec {
+  pname = "dwdwfsapi";
+  version = "1.0.3";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-PX1b1msaZH8HKVBo3GU7TOr8Lo7INLjjJBkDHHs6mzk=";
+  };
+
+  propagatedBuildInputs = [ requests ciso8601 ];
+
+  # no tests
+  doCheck = false;
```

Comment:
https://github.com/stephan192/dwdwfsapi/blob/master/tests/test_weatherwarnings.py

You might have to fetch the package from GitHub, so that we can have tests.

Reply:
All tests in that repo require network access.

Reply:
```diff
+  # All tests require network access
+  doCheck = false;
```

Reply:
Done, thanks.